docs/Tutorial_MPDist.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    plt.style.use('stumpy.mplstyle')
</pre><p> </p><p>Can you replace this line with:</p><p> </p><pre class="ql-syntax" spellcheck="false">plt.style.use('https://raw.githubusercontent.com/TDAmeritrade/stumpy/main/docs/stumpy.mplstyle')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/433/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>